src/interfaces/extensionSettings.ts

```diff
@@ -24,9 +24,19 @@ export interface ExecutionEnvironmentSettings {
   volumeMounts: Array<IVolumeMounts>;
 }
 
+export interface UserResponse {
+  rh_org_has_subscription: boolean;
+  rh_user_has_seat: boolean;
+  rh_user_is_org_admin: boolean;
+  external_username: string;
+  username: string;
+  org_telemetry_opt_out: boolean;
+}
+
 export interface LightSpeedServiceSettings {
   enabled: boolean;
   URL: string;
   suggestions: { enabled: boolean };
   model: string | undefined;
+  orgTelemetryOptOut?: boolean;
```

Comment:
This interface defines the vscode settings used for Lightspeed and those settings appear on vscode Settings UI.  Since we are not going to allow user to specify `orgTelemetryOptOut` on vscode Settings UI, I think we'd better move it to `LightSpeedAuthenticationProvider` like the way we handle `rhUserHasSeat`.

![Screenshot from 2024-02-05 10-34-57](https://github.com/ansible/vscode-ansible/assets/27698807/eb453051-098d-4e6f-ac0d-478381d49b2f)

Reply:
oh good point @TamiTakamiya , thanks!

Reply:
Great point, thank you @TamiTakamiya! Refactored. 👍